app/src/main/java/org/mozilla/focus/searchsuggestions/ui/SearchSuggestionsFragment.kt

```diff
@@ -116,7 +135,38 @@ class SearchSuggestionsFragment : Fragment(), CoroutineScope {
         }
 
         val searchSuggestionsView = view.findViewById<ComposeView>(R.id.search_suggestions_view)
-        searchSuggestionsView.setContent { SearchOverlay(searchSuggestionsViewModel) }
+
+        searchSuggestionsView.setContent {
```

Comment:
nit: Let's move all this composable code into a function so that `onViewCreated()` does not contain all that compose code?